textile/features.textile

```diff
@@ -1606,7 +1606,7 @@ h4. ClientOptions
 ** @(TO3d)@ @tls@ boolean - defaults to true. If false, will not use TLS for all connections
 ** @(TO3e)@ @autoConnect@ boolean - defaults to true. If false, suppresses the automatic initiation of a connection when the library is instantiated
 ** @(TO3f)@ @useBinaryProtocol@ boolean - defaults to true. If false, forces the library to use the JSON encoding for REST and Realtime operations, instead of the default binary msgpack encoding
-** @(TO3g)@ @queueMessages@ boolean - defaults to true. If false, suppresses the default queueing of messages when connection states that anticipate imminent connection (connecting and disconnected). Instead, publish and presence state changes will fail immediately if not in the connected state
+** @(TO3g)@ @queueMessages@ boolean - defaults to true. If false, suppresses the default queueing of publish and presence messages when connection either in the `CONNECTING` or `DISCONNECTED` states. Otherwise messages should be failed immediately if not in the `CONNECTED` state.
```

Comment:
```suggestion
** @(TO3g)@ @queueMessages@ boolean - defaults to true. If false, suppresses the default queueing of messages when connection is either in the `CONNECTING` or `DISCONNECTED` state. Otherwise they should be failed immediately if not in the `CONNECTED` state.
```
I think type of `messages` that needs to be queued is already explicitly mentioned at other places.

Reply:
But the only place where they SHOULD be mentioned is this spec =) Although they've mentioned in the end, but making it clear from the beginning makes sense I guess.

Reply:
No no, now it shows different text! Please get back mentioning type of messages.

Reply:
https://github.com/ably/specification/commit/ef10bc10db4f55191c8914889a6e1a1fe69073cc

Reply:
https://github.com/ably/specification/pull/194/commits/b1a46dd8360aa2578ec3b50aab7f6e079e178472

Reply:
See this [thread](https://github.com/ably/specification/pull/194#discussion_r1651042915)

Reply:
That thread addresses the first of the three points I made, other two are still unaddressed

Reply:
https://github.com/ably/specification/pull/194/commits/1b4c8b2ffc48a294a1a9dfb34b5f27073f2efe16 @SimonWoolf